### DISCUSSION TITLE
Welcome to radix-vue Discussions!

### DISCUSSION THREAD

**Discussion by k11q** [unovue/reka-ui#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mukundshah** [comment]
Hi @khairulhaaziq , do we have a discord server?

**Reply by zernonia** [reply]
we certainly have! 
https://chat.radix-vue.com/

**Comment by ojvribeiro** [comment]
Thanks for this such nice lib, @zernonia. Now it reached 1.0 I can wait to use it in the platform at the company I work.
